rasa/utils/tensorflow/rasa_layers.py

```diff
@@ -0,0 +1,423 @@
+import tensorflow as tf
+from typing import Text, List, Dict, Any, Union, Optional, Tuple
+import tensorflow_addons as tfa
+
+from rasa.utils.tensorflow.model_data import FeatureSignature
+from rasa.utils.tensorflow.constants import (
+    REGULARIZATION_CONSTANT,
+    WEIGHT_SPARSITY,
+    NUM_TRANSFORMER_LAYERS,
+    TRANSFORMER_SIZE,
+    NUM_HEADS,
+    UNIDIRECTIONAL_ENCODER,
+    KEY_RELATIVE_ATTENTION,
+    VALUE_RELATIVE_ATTENTION,
+    MAX_RELATIVE_POSITION,
+    HIDDEN_LAYERS_SIZES,
+    DROP_RATE,
+    SPARSE_INPUT_DROPOUT,
+    DENSE_INPUT_DROPOUT,
+    DENSE_DIMENSION,
+    CONCAT_DIMENSION,
+    DROP_RATE_ATTENTION,
+    SEQUENCE,
+    SENTENCE,
+)
+from rasa.utils.tensorflow import layers
+from rasa.shared.nlu.constants import TEXT
+
+# TODO: use this? it's in layers.py
+tfa.options.TF_ADDONS_PY_OPS = True
```

Comment:
@tabergma or @dakshvar22 do you remember, why is it here?

Reply:
@dakshvar22 ping 😉

Reply:
@samsucik thanks for the ping, i had unsubscribed from this PR :P
I don't remember completely but looking at the section [here](https://github.com/tensorflow/addons#gpu-and-cpu-custom-ops-1), it looks like it was needed because some operation wasn't working when TF tried to default to the C++ implementation.

Reply:
Alright, I think we need to make a decision here. Since the flag is originally set only in `layers.py` and nowhere else, I'd say it's related to some bits from there. And because my new layers basically factor out code from `models.py` and from other places where the flag isn't used, I'm not gonna use it in `rasa_layers.py` (of course we can add it back in if there are some issues).